### PR TITLE
Adding release-note for 0.13.2

### PR DESCRIPTION
**Pull request by maayanf24**
Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.13.2
+
+This is a bugfix release:
+
+* Stop using cluster-owned tag for ASW Security Group lookup.
```
Comment (skitt) [inline src/content/community/releases/_index.en.md]: This release also adds support for OpenShift 4.12, fixes https://github.com/submariner-io/lighthouse/pull/860, https://github.com/submariner-io/submariner/issues/2115, and includes some MTU changes (@yboaron what’s the exact status of 0.13.2? there were some backports and reverts).

**Review by skitt — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md (hunk 1 above)
Comment (maayanf24) [reply src/content/community/releases/_index.en.md]: MTU bug fix is listed for 0.14.0 release.
I'll add a note about OpenShift 4.12 support

Comment (skitt) [reply src/content/community/releases/_index.en.md]: The 0.13.2 release notes need to list _all_ the changes between 0.13.1 and 0.13.2. Readers can’t assume that changes made in 0.14.0 are also in 0.13.2.

**Review by skitt — APPROVED** [review]

File: src/content/community/releases/_index.en.md (hunk 1 above)
Comment (yboaron) [reply src/content/community/releases/_index.en.md]: 2 PRs related to MTU : 

a.  It is now possible to customize the default TCP MSS clamping value set by Submariner also for non-Globalnet deployments, covered in this PR.
b. [ fix MTU bug for OVNK](https://github.com/submariner-io/submariner/commit/9067306e47d64775ce0838fb3985a0aadbe4cd1c) , already included in 0.13.1

So, I think that the current proposal covered all MTU issues for 0.13.2

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.13.2
+
+* Added support for OpenShift 4.12.
+* Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to
+  `clusterset.local`. This prevents unnecessary retries.
+* Stop using cluster-owned tag for ASW Security Group lookup.
```
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Stop using cluster-owned tag for AWS Security Group lookup.
```

**Review by sridhargaddam — CHANGES_REQUESTED** [review]
A small typo, other wise LGTM.

**Review by yboaron — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

File: src/content/community/releases/_index.en.md [hunk 3]
```diff
@@ -4,6 +4,15 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.13.2
+
+* Added support for OpenShift 4.12.
+* Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to
+  `clusterset.local`. This prevents unnecessary retries.
+* Stop using cluster-owned tag for AWS Security Group lookup.
+* Avoid using api.ipify.org as the first resolver for public-ip.
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Avoid using api.ipify.org as the first resolver for public IPs.
```

File: src/content/community/releases/_index.en.md [hunk 4]
```diff
@@ -4,6 +4,15 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+## v0.13.2
+
+* Added support for OpenShift 4.12.
+* Service Discovery now returns a DNS error message in the response body when no matching records are found for the query to
```
Comment (dfarrell07) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Service Discovery now returns a DNS error message in the response body when no matching records are found when queried about
```

**Review by dfarrell07 — CHANGES_REQUESTED** [review]
Looks good, a few small suggestions.

File: src/content/community/releases/_index.en.md (hunk 4 above)
Comment (maayanf24) [reply src/content/community/releases/_index.en.md]: We already have this note for 0.14.0,  should I change there as well?

**Comment by nyechiel** [comment]
@dfarrell07 once this PR is merged, let's announce the release :)

File: src/content/community/releases/_index.en.md (hunk 4 above)
Comment (maayanf24) [reply src/content/community/releases/_index.en.md]: @dfarrell07 ^^

Comment (nyechiel) [reply src/content/community/releases/_index.en.md]: Yes, let's fix both for consistency.

**Review by nyechiel — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**